spring-cloud-aws-dynamodb/src/main/java/io/awspring/cloud/dynamodb/DynamoDbTableResolver.java

```diff
@@ -18,21 +18,28 @@
 import software.amazon.awssdk.enhanced.dynamodb.TableSchema;
 
 /**
- * Resolving Class and TableName to {@link TableSchema} class. Should be cached since creating {@link TableSchema} is
- * expensive.
+ * Resolving table schema and table name from a class.
  *
  * @author Matej Nedic
+ * @author Maciej Walkowiak
  * @since 3.0
  */
-public interface DynamoDbTableSchemaResolver {
+public interface DynamoDbTableResolver {
 
 	/**
-	 * Resolving Class and TableName to {@link TableSchema} class.
+	 * Resolving {@link TableSchema} from {@link Class}.
 	 *
 	 * @param clazz - the class from which table schema is resolved
-	 * @param tableName - the table name
 	 * @return table schema
 	 * @param <T> - type
 	 */
-	<T> TableSchema resolve(Class<T> clazz, String tableName);
```

Comment:
This looks like a breaking change - maybe we should keep the old classes / interfaces around for a while and deprecate them instead?

No worries if we're ok with living with the breaking changes though.